### DISCUSSION TITLE
How can I remove the weight input from the journal entry?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2754] [Q&A] [answered]
Hi

How do I remove the "Weight" entry box from the journal/today entry? I do not need that box to be in the journal page.

**Comment by sigaloid** [comment, marked as answer]
It sounds like you have left the default database in place. It contains a template for the journal entry, containing a weight attribute. If you open that note's attributes, it should have a relational attribute for a template, with the target being the template. You can click it to go to the template note, and delete it. Alternatively, you can go to the root for the journal (uppermost note containing the inheritable template attribute) and delete that there.

**Comment by gerroon** [comment]
Thanks, that worked
